Editor/EditorCore/VertexManipulationTool.cs

```diff
@@ -137,8 +138,8 @@ static void SyncPivotRotation()
         internal static Pref<bool> s_ExtrudeEdgesAsGroup = new Pref<bool>("editor.extrudeEdgesAsGroup", true);
         internal static Pref<ExtrudeMethod> s_ExtrudeMethod = new Pref<ExtrudeMethod>("editor.extrudeMethod", ExtrudeMethod.FaceNormal);
 
-        Vector3 m_HandlePosition;
-        Quaternion m_HandleRotation;
+        protected Vector3 m_HandlePosition;
```

Comment:
If I remember correctly, the coding style for protected value is thee same as public. @karl- can you confirm?